### PR TITLE
[all hosts] (Publishing) Deprecation information

### PR DESCRIPTION
**Pull request by AlexJerabek**
This PR expands our maintainability story by listing our deprecation timeline. This is consistent with the promises made by the Graph team: https://docs.microsoft.com/en-us/graph/versioning-and-support#support-policy-and-deprecation-information.

### CODE REVIEW

File: docs/publish/maintain-breaking-changes.md [hunk 1]
```diff
@@ -30,6 +30,16 @@ If you do encounter an app compatibility issue, App Assure engineers will work w
 
 To learn more about App Assure, watch [Bring your apps to Microsoft Edge with App Assure: tips and tricks](https://techcommunity.microsoft.com/t5/video-hub/bring-your-apps-to-microsoft-edge-with-app-assure-tips-and/ba-p/2167619). To submit your request for app compatibility with App Assure, complete the [Microsoft FastTrack Registration form](https://aka.ms/AppAssureRequest) or send an email to [achelp@microsoft.com](mailto:achelp@microsoft.com).
 
+## Deprecation policy
```
Comment (TristanD-MSFT) [inline docs/publish/maintain-breaking-changes.md:33]: What do you think about merging this with the office.js section above? It largely overlaps, so folding them together might read better?

Comment (TristanD-MSFT) [reply docs/publish/maintain-breaking-changes.md:33]: If we don't merge them, this says we might deprecate over 2 years, that says we generally won't, so I think we need to better align those.

**Review by TristanD-MSFT — COMMENTED** [review]
Suggest merging the section about office.js breaking changes and the deprecation policy - what do you think?

**Comment by AlexJerabek** [comment]
@TristanD-MSFT, agreed about the merge. The new version combines then a little more cleanly.

**Review by samantharamon — APPROVED** [review]
Looks good! Suggesting one fix to the Important alert.

**Comment by AlexJerabek** [comment]
Thanks @samantharamon! I'm waiting for CELA confirmation, then I'll check this in.

**Pull request merged**